server/@types/sendLegalMailApi/index.d.ts

```diff
@@ -71,6 +79,19 @@ export interface components {
       code: string
       userMessage: string
     }
+    AuthenticationRequestErrorCodes: components['schemas']['ErrorCode'] & {
```

Comment:
This is the new name for the type in the API (it was previously called `MagicLinkRequestErrorCodes`)